server.js

```diff
@@ -3,35 +3,99 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import dotenv from 'dotenv'
+import listEndpoints from 'express-list-endpoints'
+
+import data from './data/golden-globes.json'
+
+dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || `mongodb+srv://${process.env.USER_ID}:${process.env.API_KEY}@cluster0.bvwog.mongodb.net/goldenGlobe?retryWrites=true&w=majority`
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const goldenGlobeSchema = new mongoose.Schema({
+  year_film: Number,
+  year_award: Number,
+  ceremony: Number,
+  category: String,
+  nominee: String,
+  film: String,
+  win: Boolean
+})
+
+const GoldenGlobe = mongoose.model('GoldenGlobe', goldenGlobeSchema)
+
+if(process.env.RESET_DB) {
+  console.log('SEEDING')
+  const seedDB = async () => {
+    await GoldenGlobe.deleteMany()
```

Comment:
I believe we can let this console.log go, like in the song from the movie "Frozen" ❄️ (since we are talking about movies in this rep😉). Let it gooo, let it gooo.🎵